.github/workflows/test.yml

```diff
@@ -91,6 +91,18 @@ jobs:
         cache-dependency-path: pyproject.toml
     - name: Install dependencies
       run: pip install -e .[test]
+    - name: Patch /etc/hosts
+      if: runner.os != 'Windows'
+      run: |
+        echo "1.2.3.4 xn--fa-hia.de" | sudo tee -a /etc/hosts
+        echo "5.6.7.8 fass.de" | sudo tee -a /etc/hosts
+    - name: Patch C:\Windows\System32\drivers\etc\hosts
+      if: runner.os == 'Windows'
+      run: |
+        Add-Content -Path C:\Windows\System32\drivers\etc\hosts -Value "1.2.3.4 xn--fa-hia.de"
```

Comment:
windows runners have bash can we not use that?

Reply:
I don't see how that would improve the workflow. The powershell commands here are pretty clean as they are.